uhh.go

```diff
@@ -28,38 +29,67 @@ func contains(arr []string, str string) bool {
 }
 
 func (u *Uhh) Clone() error {
-	if !gitClone(u.config) {
+	if !gitClone(u.config.Repo(), u.config.LocalRepoPath()) {
 		return fmt.Errorf("Unable to clone")
 	}
 
 	return nil
 }
 
+func (u *Uhh) AddRepo(repoUrl string) error {
+    idx := len(u.config.vals.ReadRepos)
+    repoPath := path.Join(u.config.basePath, strconv.Itoa(idx))
+
+	if !gitClone(repoUrl, repoPath) {
+		return fmt.Errorf("Unable to clone")
+	}
+
+    u.config.vals.ReadRepos = append(u.config.vals.ReadRepos, strconv.Itoa(idx))
```

Comment:
Should we add the repo url here instead of the index? 

So we wouldn't know the url that the user specified and the user would not be able to edit the config by hand and add more repos.
I was thinking of a map from repo url to local repo path might be easier to read and use for us and the user.